src/server.c

```diff
@@ -5283,6 +5283,17 @@ int processCommand(client *c) {
         return C_OK;
     }
 
+    /* check if DEBUG/Module command not allowed */
+    if ((server.debug_cmd_disabled && c->cmd->proc == debugCommand) ||
+        (server.module_cmd_disabled && c->cmd->flags & CMD_MODULE)) {
+        rejectCommandFormat(c,"%s commands not allowed, you may enable it by "
+                              "editing the Redis configuration file, and setting the "
+                              "%s option to 'no', and then restarting the server.",
```

Comment:
when we add the `unixsocket`, option we may need to change part of this message. or make it more abstract.